.vscode/extensions.json

```diff
@@ -1,7 +1,6 @@
 {
   "recommendations": [
     "clinyong.vscode-css-modules",
-    "coenraads.bracket-pair-colorizer",
```

Comment:
Let's put this back.

Reply:
it's not even on the marketplace anymore

Reply:
ah so it isn't.